app/print_manager.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class PrintBook(ABC):
+    method_type = None
+
+    def __init__(self, book: Book) -> None:
+        pass
+
+    @abstractmethod
+    def print_book(self) -> None:
+        pass
+
+
+class PrintBookConsole(PrintBook):
+    method_type = "console"
+
+    def __init__(self, book: Book) -> None:
+        super().__init__(book=book)
+        self.message = f"Printing the book: {book.title}..."
+        self.content = book.content
```

Comment:
And then these variables may be redundant. Just do this logic into print_book method

Reply:
Fix in all places